source-code/core/src/config/schema.ts

```diff
@@ -57,6 +57,18 @@ export type Config = {
   experimental?: object;
 
   ideExtension?: {
+    /**
+     * Defines how message previews look like.
+     */
+    preview?: {
```

Comment:
Do you expect that people will need to customize the message preview?

Reply:
Will be removed and hard coded for now.